mix.exs

```diff
@@ -15,7 +15,6 @@ defmodule Speck.MixProject do
       deps: deps(),
       docs: docs(),
       dialyzer: [
-        ignore_warnings: "dialyzer.ignore.exs",
```

Comment:
The file wasn't removed but renamed to `.dialyzer_ignore.exs` which is the default name `dialyxir` looks for in a project. The way some of the source code is written in `dialyxir` also suggests usage of `ignore_warnings` is considered legacy. There is a bug in `dialyxir` right now which flat out ignores the `ignore_warnings` opt and so even w/ `ignore_warnings` defined it'll still look for `.dialyzer_ignore.exs` hence the rename.

Reply:
I pinned the version to `1.4.3` avoiding the issue entirely.

Reply:
✅ Hm, I wonder if `.dialyzer_ignore.exs` became a thing after I created the design pattern around it. Since that's what their docs say to use, that's a good reason for us to switch over to doing it that way. Our project template will also need to be updated.